app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CollaboratorsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @wiki = Wiki.find(params[:wiki_id])
+    @collaborator = @wiki.collaborators.new
```

Comment:
So when I previously got stuck, it was because I couldn't figure out how to pass the wiki id into my collaborator policy.  How I finally got that into the policy was by instantiating `@collaborator` through `@wiki` as seen in the line above.  Typically in the all other Bloc examples the object is instantiated just as `@collaborator`, so I wanted to make sure this wasn't a super bad practice!